src/main/java/org/opentripplanner/routing/services/notes/StreetNotesSource.java

```diff
@@ -1,14 +1,88 @@
 package org.opentripplanner.routing.services.notes;
 
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.SetMultimap;
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Set;
 import org.opentripplanner.street.model.edge.Edge;
+import org.opentripplanner.street.model.edge.TemporaryPartialStreetEdge;
+import org.opentripplanner.street.model.note.StreetNote;
 import org.opentripplanner.street.model.note.StreetNoteAndMatcher;
+import org.opentripplanner.street.model.note.StreetNoteMatcher;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
- * A source of notes for edges.
- *
- * @author laurent
+ * A notes source of static notes, created at graph building stage and not modified
+ * thereafter.
  */
-public interface StreetNotesSource {
-  Set<StreetNoteAndMatcher> getNotes(Edge edge);
+public class StreetNotesSource implements Serializable {
```

Comment:
Is the naming convention to have this named like `StreetNoteSource` instead? I tried quickly looking how we usually have this and it seemed to be like that or something like `StreetNoteDataSource`.

Reply:
I don't know and to be honest I don't really care. Let's have a quick chat in tomorrow's meeting.

Reply:
We will call it `StreetNoteModel`.